package.json

```diff
@@ -114,6 +116,7 @@
     "react-native-screens": "^3.20.0",
     "react-native-sound": "^0.11.2",
     "react-native-svg": "13.4.0",
+    "react-native-toast-notifications": "^3.3.1",
```

Comment:
we are already using the toast library from sendbird. 

`import { useToast } from '@sendbird/uikit-react-native-foundation'`

You can find examples from the repo code

Reply:
I know the uikit toast. 
New implementation of toast presented in ux design. If overlapping names are a problem I will change the name if necessary.

Reply:
can you migrate the old ones to this then? or does that require more work?